var/spack/repos/builtin/packages/libmd/package.py

```diff
@@ -11,10 +11,11 @@ class Libmd(AutotoolsPackage):
     macOS, Solaris) libraries and lacking on others like GNU systems."""
 
     homepage = "https://www.hadrons.org/software/libmd/"
-    url = "https://archive.hadrons.org/software/libmd/libmd-1.0.3.tar.xz"
+    url = "https://archive.hadrons.org/software/libmd/libmd-1.1.0.tar.xz"
```

Comment:
Spack is able to infer URLs for new versions so this doesn't need to be updated.